Cargo.toml

```diff
@@ -112,3 +112,6 @@ any_impl = []
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[patch.crates-io]
+miniz_oxide = { git = "https://github.com/Frommi/miniz_oxide", branch = "master" }
```

Comment:
The only thing left should be getting a miniz_oxide 0.8.10 release, so this patch isn't necessary for the tests to pass